### ISSUE TITLE
Display selected place types at top

### ISSUE THREAD

**Issue by neslihanturan** [commons-app/apps-android-commons#3242]
**Summary:** 

This was the buggy code I used for this purpose, but should be implemented in a more proper way. Sharing this code piece just to give the idea:
```
               displayedLabels.remove(label);
                displayedLabels.add(selectedLabels.size()-1, label);
                notifyDataSetChanged();
                smoothScroller.setTargetPosition(0);
                recyclerView.getLayoutManager().startSmoothScroll(smoothScroller);
```
![image](https://user-images.githubusercontent.com/3127881/69871008-b7831d00-12c2-11ea-93ac-9119b98fe003.png)

**Steps to reproduce:** 

When you select place types they stay where they are:
![image](https://user-images.githubusercontent.com/3127881/69870750-f4024900-12c1-11ea-8575-c3f1472d7291.png)

Ideally the list should be resorted and they should be displayed at the top, and we should make user jump to top after each selection **(this behavior is open to discussion, maybe there is no need for this improvement at all. Maybe this is not an improvement already and we should close this issue, please comment)** Here is (maybe) expected behavior:
![image](https://user-images.githubusercontent.com/3127881/69870922-6c690a00-12c2-11ea-8a20-fb7914fb8eff.png)

**Screen-shots:** 

Can be created by pressing the Volume Down and Power Button at the same time on Android 4.0 and higher.

**Would you like to work on the issue?**

I shared my buggy solution already. first we should decide if this one is really needed or not, then if it is really needed someone else can start to work on it.

**Comment by chaytanyasinha** [comment]
Yes @neslihanturan even I have seen this feature in many applications and its very good to have these features as it increases the iser experience and make it Comfortable
